src/node_config.rs

```diff
@@ -160,6 +160,25 @@ fn app() -> App<'static, 'static> {
                 .help("Load settings from saved config file. \
                     Config file format is YAML")
                 .takes_value(true)))
+        .subcommand(SubCommand::with_name("get_pk")
```

Comment:
I think hyphen'd be better: https://stackoverflow.com/questions/1253679/should-command-line-options-in-posix-style-operating-systems-be-underscore-style

Reply:
Also how about `derive` instead of `get`?